zk-token-sdk/src/instruction/transfer/mod.rs

```diff
@@ -0,0 +1,115 @@
+mod encryption;
+mod transfer_with_fee;
+mod transfer_without_fee;
```

Comment:
```suggestion
mod with_fee;
mod without_fee;
```
Wdyt?
In some of the intermediate commits, I really wanted this to avoid things like `use instruction::transfer::transfer_with_fee`. I feel less strongly about it in the end, but still might be nice.

Reply:
Yep, I think this will be much cleaner. Changed to `with_fee` and `without_fee`.